src/no/runsafe/nchat/channel/ChannelManager.java

```diff
@@ -49,14 +49,11 @@ public IChatChannel getPrivateChannel(ICommandExecutor player1, ICommandExecutor
 		if (cmp == 0)
 			return null;
 
-		if (player1 instanceof IPlayer && player2 instanceof IPlayer)
-		{
-			if (ignoreHandler.playerIsIgnoring((IPlayer) player1, (IPlayer) player2))
-				return null;
-
-			if (ignoreHandler.playerIsIgnoring((IPlayer) player2, (IPlayer) player1))
-				return null;
-		}
+		if ((player1 instanceof IPlayer && player2 instanceof IPlayer)
```

Comment:
The channel name for PMs is %<someone>-<else> to make sure nobody ever creates a normal channel with that name.